models/engine/file_storage.py

```diff
@@ -81,6 +81,6 @@ def count(self, cls=None):
         """Count the number of objects in storage"""
         count = 0
         new_dict = self.all(cls)
-        for object in new_dict.values():
+        for object_all in new_dict.values():
```

Comment:
As in the other file, it is appreciated the change of name of the variable because it lets what is obtained in that 'for' is the complete object.